doc/modules/mixture.rst

```diff
@@ -135,6 +135,42 @@ parameters to maximize the likelihood of the data given those
 assignments. Repeating this process is guaranteed to always converge
 to a local optimum.
 
+Choice of the Initialization Method
+-----------------------------------
+
+There is a choice of four initialization methods (as well as inputting user defined
+initial means) to generate the initial centers for the model components: 
+
+k-means (default)
+  This applies a traditional k-means clustering algorithm. This can be computationally expensive compared to other initialization methods.
```

Comment:
```suggestion
k-means (default)
  This applies a traditional k-means clustering algorithm.
  This can be computationally expensive compared to other initialization methods.
```

Reply:
Thanks to you for your valuable feedback, this has been quite the learning opportunity for us :)